tasks/update-ocp-tag/update-ocp-tag.yaml

```diff
@@ -47,15 +47,33 @@ spec:
 
         # Function to replace tag in an image
         replace_tag() {
```

Comment:
It would be nice to specify the expected args of the function.

Reply:
done.